squidb/src/com/yahoo/squidb/sql/SqlUtils.java

```diff
@@ -7,7 +7,6 @@
 
 import android.text.TextUtils;
 
-import java.math.BigDecimal;
```

Comment:
It was handled inconsistently. We'd inline it as a String in decimal form, but DatabaseUtils would bind it as a Long (since it subclasses Number). I think it's better to stick with the DatabaseUtils behavior for consistency, unless we want to reimplement DatabaseUtils ourself. Since we'd just coerce to a potentially-lower-precision double anyways though, it makes more sense for users to use Double or AtomicReference<Double>